### ISSUE TITLE
Name attribute considered obsolete on anchor tags (Beta)

### ISSUE THREAD

**Issue by marigerr** [freeCodeCamp/freeCodeCamp#15672]
#### Challenge Name

http://beta.freecodecamp.com/en/challenges/basic-html-and-html5/link-to-internal-sections-of-a-page-with-anchor-elements

#### Issue Description

Above lesson,  instructs to use `name` attribute on the target anchor tag.  However I came across a few references that state that using `name` attribute on anchor tags is considered obsolete and that `id` tag should be used instead.  

From [W3C obsolete features](https://www.w3.org/TR/html5/obsolete.html#obsolete)

> Authors should not specify the name attribute on \<a\> elements. If the attribute is present, its value must not be the empty string and must neither be equal to the value of any of the IDs in the element's home subtree other than the element's own ID, if any, nor be equal to the value of any of the other name attributes on \<a\> elements in the element's home subtree. If this attribute is present and the element has an ID, then the attribute's value must be equal to the element's ID. In earlier versions of the language, this attribute was intended as a way to specify possible targets for fragment identifiers in URLs. The id attribute should be used instead.

It might be useful to update lesson to also include that you can use the `id` of an existing tag to jump to, for example:  
```
<a href="#target">Jump to Some Heading</a>
<h2 id="target">Some Heading</h2>
```
If approved as an issue, I'd love to submit a PR to update the lesson and test.    
Thanks for the great site and community!

**Comment by dhcodes** [comment]
I say go for it!

**Comment by marigerr** [comment]
ok great will do! :)

**Comment by dhcodes** [comment]
Closed by PR #15682
